tokenManager.js

```diff
@@ -75,36 +80,38 @@ function removeExpiredTokens() {
 function tokenMiddleware(req, res, next) {
```

Comment:
This function should probably be called `authenticationMiddleware()` (or something like that) to reflect that it doesn't just deal with JavaScript Web Tokens any more.